### PR TITLE
fix(logicaldatabase): several bugfixes when extacting logical databases and showing the databases

### LINKED ISSUE #3544: [Bug]: In logical table management, data source information is missing
### ODC version

4.3.2

### OB version

no related 

### What happened?

image
image
In logical table management, data source information is missing，
In the logical topology, the data source should also be displayed
image
image

### What did you expect to happen?

In Logical Table Management, select a logical database, expand it, and view the field column, which displays the name of the corresponding data source

In the logical topology, the data source should also be displayed

### How can we reproduce it (as minimally and precisely as possible)?

In Logical Table Management, select a logical database, expand it, and view the field columns, which contain missing data source information

### Anything else we need to know?

_No response_

### PR DESCRIPTION
**Pull request by MarkPotato777**
#### What type of PR is this?

type-bug
#### What this PR does / why we need it:
1. Exclude the physical tables that have already belonged to a logical table.
2. Ignore those logical tables that have only one physical table.
3. Show more details for the front-end in the API
#### Which issue(s) this PR fixes:

Fixes #3544 #3564 

#### Additional documentation e.g., usage docs, etc.:

```docs

```

### CODE REVIEW

File: server/odc-service/src/main/java/com/oceanbase/odc/service/connection/logicaldatabase/LogicalTableExtractTask.java [hunk 1]
```diff
@@ -128,31 +131,34 @@ public void run() {
...
 
-            logicalTables.stream().filter(table -> !existedTables.contains(table.getName())).forEach(table -> {
-                DBObjectEntity tableEntity = new DBObjectEntity();
-                tableEntity.setDatabaseId(logicalDatabase.getId());
-                tableEntity.setType(DBObjectType.LOGICAL_TABLE);
-                tableEntity.setName(table.getName());
-                tableEntity.setOrganizationId(logicalDatabase.getOrganizationId());
-                DBObjectEntity savedTableEntity = dbObjectRepository.save(tableEntity);
+            logicalTables.stream()
+                    .filter(table -> !existedTables.contains(table.getName()) && table.getActualDataNodes().size() > 1)
```
Comment (MarkPotato777) [inline server/odc-service/src/main/java/com/oceanbase/odc/service/connection/logicaldatabase/LogicalTableExtractTask.java:135]: only process the logical table that have more than 1 physical tables

**Review by yizhouxw — APPROVED** [review]
lgtm

**Review by yhilmare — APPROVED** [review]
lgtm

**Review by yizhouxw — APPROVED** [review]

**Review by yhilmare — APPROVED** [review]
lgtm

**Pull request merged**